clients/admin-ui/cypress/e2e/config-wizard.cy.ts

```diff
@@ -82,5 +83,26 @@ describe("Config Wizard", () => {
       cy.getByTestId("generic-msg");
       cy.getByTestId("error-log").contains("Internal Server Error");
     });
+
+    it("Allows stepping back to the previous step during an in-progress scan", () => {
+      cy.intercept(
+        "POST",
+        "/api/v1/generate",
+
+        (req) => {
+          req.continue((res) => {
```

Comment:
I'm going to test this out, but I thought `continue` would send the request to the actual server.